test/e2e/framework/framework.go

```diff
@@ -200,26 +202,29 @@ func (f *Framework) setupNamespace(ctx context.Context) {
 	userSA, err = WaitForServiceAccount(ctxUserSa, f.KubeAdminClient().CoreV1(), userSA.Namespace, userSA.Name)
```

Comment:
Previously we waited until token was added in SA.Secrets, now we wait until secret is populated, so this watch is no longer needed.